modular_skyrat/master_files/code/modules/mob_spawn/mob_spawn.dm

```diff
@@ -22,6 +22,9 @@
 	if(loadout_enabled)
 		spawned_human.equip_outfit_and_loadout(outfit, spawned_human.client.prefs)
 
+	if(!loadout_enabled)
```

Comment:
```suggestion
	else
```

Reply:
Wanted to err on the side of caution on else. Should be popping up soon.